assets/python-nemo/internxt-virtual-drive.py

```
@@ -0,0 +1,255 @@
+
+
+    def _get_availability(self, file):
+      base64_encoded = self._encode_file_path(file)
+
+      if file.is_directory() :
+        url = base_url + 'folders/' + base64_encoded
+      else :
+        url = base_url + 'files/' + base64_encoded
+
+
+      response = requests.get(url)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)